package.json

```diff
@@ -39,23 +39,28 @@
   "author": "Andela Simulations Programme",
   "license": "MIT",
   "dependencies": {
+    "body-parser": "^1.18.3",
     "cors": "^2.8.4",
     "express": "^4.16.3",
     "express-jwt": "^5.3.1",
```

Comment:
Why do you need `express-jwt` and `jsonwebtoken` seem like they do the same thing?